dev-scripts/lagoon-set-gh-deploy-status.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Ensure the required arguments are passed
+if [ "$#" -lt 1 ] || [ "$#" -gt 2 ]; then
+  echo "Usage: $0 <state> [optional_environment_url]"
+  exit 1
+fi
+
+# Assign the state passed as an argument
+STATE=$1
+DRUPAL_URL=${2:-""}
+
+GH_DEPLOYMENT_ID=$(cat /tmp/GH_DEPLOYMENT_ID)
+LAGOON_DEPLOYS_LOG_URL=$(cat /tmp/LAGOON_DEPLOYS_LOG_URL)
+
+echo "Setting GH deployment status '$GH_DEPLOYMENT_ID': '$STATE'"
+
+# GitHub API request to update deployment status
+curl -L -X POST -H "Authorization: Bearer $GH_DEPLOYMENT_TOKEN" \
+  "https://api.github.com/repos/danskernesdigitalebibliotek/dpl-cms/deployments/$GH_DEPLOYMENT_ID/statuses" \
```

Comment:
I think the start of the deployments url "https://ui.lagoon.dplplat01.dpl.reload.dk/projects" should be an argument